test/Conversion/FIRRTLToHW/lower-to-hw.mlir

```diff
@@ -9,6 +9,43 @@ firrtl.circuit "Simple"   attributes {annotations = [{class =
 "sifive.enterprise.firrtl.ExtractCoverageAnnotation", directory = "dir2",  filename = "./dir2/filename2" }, {class =
 "sifive.enterprise.firrtl.ExtractAssertionsAnnotation", directory = "dir3",  filename = "./dir3/filename3" }]}
 {
+  // Headers
+  // CHECK:      sv.ifdef  "PRINTF_COND_" {
```

Comment:
does this emit as a literal ifdef, else? Or will it emit as an ifndef? IT seems like the latter would be nicer?

Reply:
Good question. This will become `ifndef`. There is no actual representation for this, but `sv.ifdef <cond> {} else { ... }` will emit as `ifndef <cond>`.